vulkaninfo/CMakeLists.txt

```diff
@@ -57,21 +63,21 @@ if(UNIX AND NOT APPLE) # i.e. Linux
         find_package(XCB REQUIRED)
         target_include_directories(vulkaninfo PRIVATE ${XCB_INCLUDE_DIRS})
         target_link_libraries(vulkaninfo ${XCB_LIBRARIES})
-        target_compile_definitions(vulkaninfo PRIVATE -DVK_USE_PLATFORM_XCB_KHR)
+        target_compile_definitions(vulkaninfo PRIVATE -DVK_USE_PLATFORM_XCB_KHR -DVK_NO_PROTOTYPES)
```

Comment:
Is this (and below) part of using volk, or some other maintenance?

Reply:
`DVK_NO_PROTOTYPES` is whats needed for stopping the headers from adding the function prototypes. This is required so vulkaninfo uses the function pointers provided by volk.

Reply:
ok